packages/core/src/new-api/internal/execution/execution-engine.ts

```diff
@@ -175,7 +198,9 @@ export class ExecutionEngine {
           // status: ExecutionStatus.STARTED,
           dependencies: [...future.dependencies].map((f) => f.id),
           // history: [],
-          storedArtifactPath: "./artifact.json",
+          storedArtifactPath: await artifactResolver.resolvePath(
```

Comment:
This will change drastically with the deployment loader introduction in https://github.com/NomicFoundation/ignition/issues/109. But to get different values in tests I needed a minimal path resolution.